lib/beautiful/init.lua

```diff
@@ -117,14 +117,25 @@ function beautiful.init(config)
         if type(config) == 'string' then
             -- Expand the '~' $HOME shortcut
             config = config:gsub("^~/", homedir .. "/")
+            configdir = config:gsub("/[^/]+$", "")
```

Comment:
Please avoid global variables. This is bad and this causes the Travis CI to fail.